drivers/docker/utils.go

```diff
@@ -218,3 +220,52 @@ func isParentPath(parent, path string) bool {
 	rel, err := filepath.Rel(parent, path)
 	return err == nil && !strings.HasPrefix(rel, "..")
 }
+
+func parseVolumeSpec(volBind, os string) (hostPath string, containerPath string, mode string, err error) {
```

Comment:
this is a lil bit nasty, but it'll be the least effort to switching based on container.OSType so 👍

Reply:
Yeah, the intention is to pass `containerOS`, will rename parameter.  The caller now passes `runtime.GOOS` with a comment that we should switch to container OS when we support linux container on Windows.  Will clarify comment and parameter name before merging.